http-endpoint/src/lib.rs

```diff
@@ -107,27 +115,46 @@ pub async fn run(config: Config, startup: &mut dyn Startup) -> anyhow::Result<()
     }
 
     let main = HttpBuilder::new(config.http, Some(startup.runtime_config()), move |cfg| {
+        let mut cors = Cors::default()
```

Comment:
So you could to an `let cors: Cors = config.http.cors.into();'`

Reply:
Yes, the `into` needs to provide a few additional settings, like the code below that comment.